test.sh

```diff
@@ -58,6 +58,23 @@ cd $REPO_ROOT_DIR
 
 # Get a CCM cluster if not already configured (see available settings in dcos-commons/tools/README.md):
 if [ -z "$CLUSTER_URL" ]; then
+    if [ "$SECURITY" = "strict" ]; then
```

Comment:
The CLI is also installed in `run_tests.py`.  That functionality needs to be factored out, not duplicated.